nipy/algorithms/graph/tests/test_forest.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
+
```

Comment:
Are you asking if `from __future__ import print_function` is necessary?   Yes, I believe so...